### PR TITLE
NMS-14065: Document missing handlers

### PR DESCRIPTION
**Pull request by Bonrob2**
### All Contributors

* [ ] Have you read our [Contribution Guidelines](https://github.com/OpenNMS/opennms/blob/develop/CONTRIBUTING.md)?
* [ ] Have you (electronically) signed the [OpenNMS Contributor Agreement](https://cla-assistant.io/OpenNMS/opennms)?

### Contribution Checklist

* Please [make an issue in the OpenNMS issue tracker](https://issues.opennms.org) if there isn't one already.<br />Once there is an issue, please:
  1. update the title of this PR to be in the format: `${JIRA-ISSUE-NUMBER}: subject of pull request`
  2. update the JIRA link at the bottom of this comment to refer to the real issue number
  3. prefix your commit messages with the issue number, if possible
  4. once you've created this PR, please link to it in a comment in the JIRA issue
  Don't worry if this sounds like a lot, we can help you get things set up properly.
* If this is a new or updated feature, is there documentation for the new behavior?
* If this is new code, are there unit and/or integration tests?
* If this PR targets a `foundation-*` branch, does it try to avoid changing files in `$OPENNMS_HOME/etc/`?

### What's Next?

A PR should be assigned at least 2 reviewers.  If you know that someone would be a good person to review your code, feel free to add them.

If you need help making additions or changes to the documentation related to your changes, please let us know.

In any case, if anything is unclear or you want help getting your PR ready for merge, please don't hesitate to say something in the comments here,
or in [the #opennms-development chat channel](https://chat.opennms.com/opennms/channels/opennms-development).

Once reviewer(s) accept the PR and the branch passes continuous integration, the PR is eligible for merge.

At that time, if you have commit access (are an OpenNMS Group employee or a member of the OGP) you are welcome to merge the PR when you're ready.
Otherwise, a reviewer can merge it for you.

Thanks for taking time to contribute!

### External References

* JIRA (Issue Tracker): http://issues.opennms.org/browse/${JIRA-ISSUE-NUMBER}

### CODE REVIEW

**Comment by Bonrob2** [comment]
@jeffgdotorg , documentation based on what you wrote in the confluence document. The parameters aren't documented in the various plugin repos ... that would have to come under separate Jiras. Let me know if this is sufficient. 
@mmahacek , I'll have you review the non-draft PR, as there might be a lot of changes. :-)

File: docs/modules/reference/pages/provisioning/handlers/http.adoc [hunk 1]
```diff
@@ -1,8 +1,8 @@
 
 [[http-handler]]
-= HTTP Handler
+= HTTP(S) Handler
 
-The HTTP handler imports a properly-formatted requisition definition from an XML file stored remotely on a web server.
+The HTTP(S) handler imports a properly-formatted requisition definition from an XML file stored remotely on a web server.
```
Comment (jeffgdotorg) [inline docs/modules/reference/pages/provisioning/handlers/http.adoc:5]: Can we say "from an XML document served by a remote web server"?

File: docs/modules/reference/pages/provisioning/handlers/requisition-plugin.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
+
+[[requisition-plugin]]
+= Requisition Plugins
+
+{page-component-title} provides plugins that can provision nodes from additional platforms, including software-defined networking, hypervisor, and configuration management database (CMDB) platforms.
+
+Each of these plugins has a handler that invokes the plugin with default settings, and imports requisitions generated by that plugin.
```
Comment (jeffgdotorg) [inline docs/modules/reference/pages/provisioning/handlers/requisition-plugin.adoc:7]: Each of these plugins has a handler that invokes the plugin with default settings.
Some plugins accept options in the form of query parameters, which shape the requisitions generated by that plugin and imported by Provisiond.

**Review by jeffgdotorg — CHANGES_REQUESTED** [review]
See review comments.

**Comment by Bonrob2** [comment]
@indigo423 , since Mark is away, could you review? I've updated based on Jeff's feedback.

**Review by mmahacek — COMMENTED** [review]
There is talk of "plugins". Are those the http/VMware handlers or something else?

**Comment by Bonrob2** [comment]
> There is talk of "plugins". Are those the http/VMware handlers or something else?

The plugins are OpenDaylight, ACI, PRIS. But since they are experimental, Jeff suggested not documenting them, but having the "Requisition Plugins" topic for those who might know about them. We could consider linking to them in GitHub.

**Comment by mmahacek** [comment]
Personally, if there aren't links to them, I'd leave the mention of them out. As a user, it's confusing in the current state to not have anything describing what plugins are or that they're currently experimental. Should probably at least mention that fact.

**Comment by Bonrob2** [comment]
Given that we have an entire set of documentation for "nephron" which is also experimental, I'm inclined to link to them in GitHub.

**Review by mmahacek — APPROVED** [review]

**Pull request merged**